src/docker/Dockerfile

```diff
@@ -49,45 +46,26 @@ RUN apt-get update && apt-get install -y \
     bsdmainutils \
     libxml2-utils \
     build-essential \
-    automake autoconf
+    automake autoconf \
+    git \
+    s3cmd \
+    libmagic-dev=1:5.14-2ubuntu3.3 \
+    zlib1g-dev=1:1.2.8.dfsg-1ubuntu1 \
+    ruby2.2 \
+    ruby2.2-dev \
+    openjdk-7-jdk="${OPENJDK_VERSION}"
```

Comment:
@amihaiemil according to https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/, we should do ` && rm -rf /var/lib/apt/lists/*` after each `apt-get` block